lib/restAPIs/userManagement.js

```diff
@@ -4,13 +4,20 @@ const assertParam = require('../util/assertParam');
 const RestUtil = require('../util/RestUtil');
 const buildCommonQuery = require('../util/buildCommonQueryParams');
 
-
 module.exports = class RestTopic extends RestUtil {
-  _getBase(){
+  _getBase() {
     return 'users';
   }
 
-  getUsers(nameBegins, abbreviationBegins, offset = 0, limit = -1, fields){
+  /**
+   * @public Get information for a specific set of users.
+   * @param {String} nameBegins - Characters that the user name must begin with
+   * @param {String} abbreviationBegins - Characters that the user abbreviation must begin with
+   * @param {int} offset - Used to control paging behaviour
+   * @param {int} limit - Maximum number of items returned
+   * @param {String} fields - Top level fields to be included in the response
```

Comment:
`String` -> `string`

Also in JS `int` should be `number`

Reply:
Done